### PR TITLE
Xandor

### PR DESCRIPTION
**Pull request by XandorMaker**

### CODE REVIEW

File: WlanManager.cs [hunk 1]
```diff
@@ -42,14 +54,12 @@ public void CreateProfiles()
             {
                 foreach (string path in pathes)
                 {
-                    List<string> wlans = wlanList; //wlans, это должно быть название массива а вот wlanList это сама переменная со списком
-                    wlanList.Add(Path.GetFileNameWithoutExtension(path));
```
Comment (postwj) [inline WlanManager.cs:46]: please fix formatting

**Review by postwj — DISMISSED** [review]
Please see comments

File: WlanManager.cs (hunk 1 above)
Comment (XandorMaker) [reply WlanManager.cs:46]: Ok Bro

Comment (postwj) [reply WlanManager.cs:46]: please do it

Comment (XandorMaker) [reply WlanManager.cs:46]: XD can we continue pls_

**Review by XandorMaker — COMMENTED** [review]
All is fixed

**Review by postwj — APPROVED** [review]

**Pull request merged**